examples/step-19/step-19.cc

```diff
@@ -662,21 +665,17 @@ namespace Step19
           for (const auto &particle : particles_in_cell)
             particle_positions.push_back(particle.get_reference_location());
 
-          const Quadrature<dim> quadrature_formula(particle_positions);
-          FEValues<dim>         particle_position_fe_values(mapping,
```

Comment:
You now no longer use the `FEValues` object, but the documentation of the function continues to describe this approach. Can you update?

Reply:
Done!